### PR TITLE
feature: tree based metrics for explainability

### PR DESCRIPTION
**Pull request by Kleyt0n**

### CODE REVIEW

**Comment by crismunoz** [comment]
- add explainability/metrics/tree/tree_explainability_metric 
- add test functions

File: docs/source/reference/explainability/metrics.rst [hunk 1]
```diff
@@ -25,5 +25,18 @@
     :nosignatures:
     :template: function.rst
     :toctree: .generated/
+    
+    data_stability
```
Comment (crismunoz) [inline docs/source/reference/explainability/metrics.rst]: i removed data stability

File: tutorials/explainability/demos/explainable_clustering.ipynb
Comment (crismunoz) [inline tutorials/explainability/demos/explainable_clustering.ipynb:1]: remove

File: docs/source/reference/explainability/metrics.rst (hunk 1 above)
Comment (kleyt0n) [reply docs/source/reference/explainability/metrics.rst]: done

**Review by crismunoz — APPROVED** [review]

**Pull request merged**